src/applications/accredited-representative-portal/tests/unit/containers/GetHelpPage.unit.spec.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { expect } from 'chai';
+import GetHelpPage from '../../../containers/GetHelpPage';
+import { renderTestApp } from '../helpers';
+
+describe('Get Help Page', () => {
+  it('renders get help heading and content', () => {
+    const { getByTestId } = renderTestApp(<GetHelpPage />);
+    expect(getByTestId('get-help-page-heading').textContent).to.eq(
+      'Get help using the portal',
+    );
+    expect(getByTestId('get-help-page-content').textContent).to.eq(
+      'You can email the portal team for help at RepresentativePortalHelp@va.gov. We monitor the email Monday through Friday from 8am to 4pm ET. Someone from the team will respond to your email within 1-2 business days.',
```

Comment:
Checking for exact text at this level heightens the chances of this test failing due to a simple content update in the future. The verification that this text matches exactly doesn't really provide a lot of information about the stability or functionality of the page. I'd recommend more simply checking if the element loads and contains text as expected, as opposed to checking for an exact match on text. The way it's currently set up, means the content and this test would always have to be updated in-sync with each other, which makes the test really have no meaning other than as a busy task while making updates.